modules/cluster-logging-release-notes-5.2.z.adoc

```diff
@@ -1,4 +1,50 @@
 //Z-stream Release Notes by Version
+[id="cluster-logging-release-notes-5-2-8"]
+= OpenShift Logging 5.2.8
+
+This release includes link:https://access.redhat.com/errata/RHSA-2022:0728[RHSA-2022:0728 OpenShift Logging Bug Fix Release 5.2.8]
+
+[id="openshift-logging-5-2-8-bug-fixes"]
+== Bug fixes
+* Before this update, the logging link in OpenShift WebConsole remained after the removal of an instance of OpenShift Logging. With this update, the removal or uninstallation of logging also removes the link. (link:https://issues.redhat.com/browse/LOG-2180[LOG-2180])
+
+[id="openshift-logging-5-2-8-CVEs"]
+== CVEs
+.Click to expand CVEs
+[%collapsible]
+====
+* link:https://access.redhat.com/security/cve/CVE-2020-28491[CVE-2020-28491]
+** link:https://bugzilla.redhat.com/show_bug.cgi?id=1930423[BZ-1930423]
+* link:https://access.redhat.com/security/cve/CVE-2022-0552[CVE-2022-0552]
```

Comment:
I don't have permissions to view this page.

Reply:
Tested in an alternate browser not logged in, displayed properly.